### DISCUSSION TITLE
Add pooling option for supabase client-js

### DISCUSSION THREAD

**Discussion by pencilcheck** [supabase/supabase#9834] [Questions] [answered]
# Feature request

https://supabase.com/docs/reference/javascript/initializing

Wonder if possible to add options to use database pooling connection for serverless functions that connects via supabase-js??

## Is your feature request related to a problem? Please describe.

A clear and concise description of what you want and what your use case is.

**Comment by steve-chavez** [comment, marked as answer]
`supabase-js` connects to the REST interface to PostgreSQL(PostgREST), it already has a connection pool.

This clarification will be added soon to the docs https://supabase.com/docs/guides/database/connecting-to-postgres#api-vs-direct-vs-pooling
